docs/docs/how-to/cloud/adding-a-custom-domain.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Adding a Custom Domain to your site on Gatsby Cloud
+description: Learn how to add a custom domain to your site on Gatsby Cloud
+---
+
+Before you can add a custom domain, you'll need to be hosting with Gatsby Cloud. If you haven't already, [turn on Gatsby Hosting](https://support.gatsbyjs.com/hc/en-us/articles/360061641114).
+
+ 
+
+## 1. Add a Domain
+If you just enabled Gatsby Hosting you should already be in the right place. Otherwise, go to **Site Settings > Hosting**. Now, click **Add Domain** above the domain list to start the process
+
+![Click Add Domain](../../images/mceclip8.png) 
```

Comment:
Two problems here:

- The filename `mceclip8` is bad, it shouldn't be cryptic but descriptive like the other ones
- Alt text could be improved

The latter applies to any alt text of images that was added here. See: https://webaim.org/techniques/alttext/

But this can be hard to do correct and it relies on the context given around the image. I do think that we give a good enough description before the image on how to achieve it, so the image is purely decorative and we can have an empty alt text:

```suggestion
![GATSBY_EMPTY_ALT](../../images/mceclip8.png) 
```